chisel/src/driver.rs

```diff
@@ -184,6 +205,278 @@ impl ChiselDriver {
+                    }
+                } else {
+                    chisel_debug!(1, "remapimports missing field 'preset'");
+                    return Err(DriverError::MissingRequiredField(
+                        name.clone(),
+                        "preset".to_string(),
+                    ));
+                }
+            }
+            "trimstartfunc" => {
+                // NOTE: preset "ewasm" maps to the default and only mode. Fixing
+                // later.
+                let trimstartfunc = TrimStartFunc::with_preset("ewasm").expect("Should not fail");
+                let module_result = trimstartfunc.translate_inplace(wasm);
+                ModuleResult::Translator(name.clone(), module_result)
+            }
+            "verifyexports" => {
+                if let Some(preset) = module.options().get("preset") {
+                    let verifyexports = VerifyExports::with_preset(preset.as_str());
+                    if let Ok(verifyexports) = verifyexports {
+                        // TODO: try other translate method as well as well
+                        let module_result = verifyexports.validate(wasm);
+                        ModuleResult::Validator(name.clone(), module_result)
+                    } else {
+                        chisel_debug!(1, "verifyexports given invalid preset");
+                        return Err(DriverError::InvalidField(
+                            name.clone(),
+                            "preset".to_string(),
+                        ));
+                    }
+                } else {
+                    chisel_debug!(1, "verifyexports missing field 'preset'");
+                    return Err(DriverError::MissingRequiredField(
+                        name.clone(),
+                        "preset".to_string(),
+                    ));
+                }
+            }
+            "verifyimports" => {
+                if let Some(preset) = module.options().get("preset") {
+                    let verifyimports = VerifyImports::with_preset(preset.as_str());
+                    if let Ok(verifyimports) = verifyimports {
+                        // TODO: try other translate method as well as well
+                        let module_result = verifyimports.validate(&wasm);
+                        ModuleResult::Validator(name.clone(), module_result)
+                    } else {
+                        chisel_debug!(1, "verifyimports given invalid preset");
+                        return Err(DriverError::InvalidField(
+                            name.clone(),
+                            "preset".to_string(),
+                        ));
+                    }
+                } else {
+                    chisel_debug!(1, "verifyimports missing field 'preset'");
+                    return Err(DriverError::MissingRequiredField(
+                        name.clone(),
+                        "preset".to_string(),
+                    ));
+                }
+            }
```

Comment:
Please add `dropnames` and `binaryenopt` though for the moment.

Reply:
We have a module called `dropnames`?

Reply:
No, but we have a case in the current CLI called `dropnames`.